From 1ae7a7c9785389d9f640ad1471c571a44b4cda7b Mon Sep 17 00:00:00 2001
From: Jean Trivelly <jean.trivelly@intel.com>
Date: Mon, 7 Jan 2013 18:22:28 +0100
Subject: [PORT FROM MAIN] wifi: modify reply length for DRIVER cmd

BZ: 75654

Broadcom driver does not support a so big buffer.
It is needed to reduce it for DRIVER command.

Feature   : Wlan
Category  : Improvement
Purpose   : AOSP+
Origin    : Upstream
Upstream-candidate: no

Change-Id: If5ae06fe3313ac8153cec18ea97e900f0c557625
Orig-Change-Id: Ie259429074d38fe80887bf454f3f30af516d4458
Signed-off-by: Jean Trivelly <jean.trivelly@intel.com>
---
 wpa_supplicant/ctrl_iface.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wpa_supplicant/ctrl_iface.c b/wpa_supplicant/ctrl_iface.c
index 171b427..35769f8 100644
--- a/wpa_supplicant/ctrl_iface.c
+++ b/wpa_supplicant/ctrl_iface.c
@@ -5570,7 +5570,7 @@ char * wpa_supplicant_ctrl_iface_process(struct wpa_supplicant *wpa_s,
 #ifdef ANDROID
 	} else if (os_strncmp(buf, "DRIVER ", 7) == 0) {
 		reply_len = wpa_supplicant_driver_cmd(wpa_s, buf + 7, reply,
-						      reply_size);
+						      4096);
 #endif
 	} else if (os_strcmp(buf, "REAUTHENTICATE") == 0) {
 		pmksa_cache_clear_current(wpa_s->wpa);
-- 
1.7.4.1

